library/src/lib/popover/popover.component.scss

```diff
@@ -1,6 +1,7 @@
 .fd-popover-custom {
     margin-right: 0;
     display: inline-block;
+    position: relative;
```

Comment:
Why do you need this? Is this how the styles work with the localization editor? If so that's very weird to me and we might want to change that in styles eventually

Reply:
We dont need it anymore. instead I just changed option `[fillControl]` on popover.